src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -218,6 +223,8 @@ private void ExitTour()
                 {
                     tmpStep.StepClosed -= Popup_StepClosed;
                 }
+                Logging.Analytics.TrackEvent(Logging.Actions.End, Logging.Categories.GuidedTourOperations, currentGuide.GuideNameResource, currentGuide.SequenceOrder);
```

Comment:
Is `End` different than `Completed`?

Reply:
yes, end is when a user closes the tour before finishing it by clicking on the 'X' button